packages/front-end/components/Modal.tsx

```diff
@@ -223,11 +225,12 @@ const Modal: FC<ModalProps> = ({
           children
         )}
       </div>
-      {submit ||
-      secondaryCTA ||
-      tertiaryCTA ||
-      backCTA ||
-      (close && includeCloseCta) ? (
+      {!hideCta &&
+      (submit ||
```

Comment:
Alternatively, change the logic in the setup modal to pass submit as `null`, which avoids needing to add a new prop here.

Reply:
Submit is required.  Even if I made it optional a SecondaryCTA when onSkip is set.  I could also null that when the step is 2 or more, but having the extra variable seemed clearer what we mean to have happen on that last page.